### PR TITLE
isort io

### PR DESCRIPTION
**Pull request by nstarman**
Signed-off-by: Nathaniel Starkman (@nstarman) <nstarkman@protonmail.com>

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label. Codestyle issues can be fixed by the [bot](https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#pre-commit).
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is this a big PR that makes a "What's new?" entry worthwhile and if so, is (1) a "what's new" entry included in this PR and (2) the "whatsnew-needed" label applied?
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

**Review by taldcroft — APPROVED** [review]
Looks good to me. I scanned the whole list and saw only changes related to import ordering or whitespace following imports (and of course pyproject.toml that looks good).

**Review by WilliamJamieson — APPROVED** [review]

**Review by saimn — APPROVED** [review]
LGTM for `io.fits`.

File: astropy/io/votable/tree.py [hunk 1]
```diff
@@ -1,37 +1,34 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 # TODO: Test FITS parsing
 
+import base64
+import codecs
+import gzip
 # STDLIB
```
Comment (pllim) [inline astropy/io/votable/tree.py:7]:
```suggestion
# STDLIB
import base64
import codecs
import gzip
```

File: astropy/io/votable/tree.py [hunk 2]
```diff
@@ -1,37 +1,34 @@
...
-import codecs
 import urllib.request
 import warnings
 
 # THIRD-PARTY
 import numpy as np
 from numpy import ma
 
+from astropy import __version__ as astropy_version
 # LOCAL
```
Comment (pllim) [inline astropy/io/votable/tree.py:18]: Hmm why is this checking astropy version? Refactoring for another day lol.
```suggestion
# LOCAL
from astropy import __version__ as astropy_version
```

**Review by matteobachetti — APPROVED** [review]
io.misc looks fine

File: astropy/io/votable/tree.py (hunk 2 above)
Comment (nstarman) [reply astropy/io/votable/tree.py]: 👍 . Thanks for the detailed check!

File: astropy/io/votable/tree.py (hunk 1 above)
Comment (nstarman) [reply astropy/io/votable/tree.py]: 👍 . Thanks for the detailed check!

**Comment by nstarman** [comment]
Ammended to include @pllim's corrections.

**Review by pllim — APPROVED** [review]
✔️ for `io.votable`

Thanks!

**Pull request merged**